evil-collection-vdiff.el

```diff
@@ -32,26 +32,34 @@
 
 (defun evil-collection-vdiff-setup ()
   "Set up `evil' bindings for `vdiff-mode'."
-  (evil-define-minor-mode-key 'normal 'vdiff-mode
-    "]c" 'vdiff-next-hunk
-    "[c" 'vdiff-previous-hunk)
-
-  ;; define `do' (diff obtain) and `dp' (diff put) bindings
-  (evil-define-minor-mode-key 'operator 'vdiff-mode
-    "o" '(menu-item
-          ""
-          nil
-          :filter (lambda (&optional _)
-                    (when (memq evil-this-operator
-                                evil-collection-delete-operators)
-                      #'vdiff-receive-changes)))
-    "p" '(menu-item
-          ""
-          nil
-          :filter (lambda (&optional _)
-                    (when (memq evil-this-operator
-                                evil-collection-delete-operators)
-                      #'vdiff-send-changes)))))
+
+  (dolist (mode '(vdiff-mode vdiff-3way-mode))
+    (evil-define-minor-mode-key 'normal mode
+      "zc" 'vdiff-close-fold
+      "zM" 'vdiff-close-all-folds
+      "zo" 'vdiff-open-fold
+      "zR" 'vdiff-open-all-folds
+      "go" 'vdiff-receive-changes
```

Comment:
I wonder if go and gp are good keys for these two commands. Maybe others can comment.

Otherwise, they are acceptable to me.

Reply:
it was copy/pasted from the readme: https://github.com/justbur/emacs-vdiff#saving-updating-and-exiting

the key improvement is making sure that both `vdiff` and `vdiff-3way` are mapped, as the latter is used by default for magit diffs.

Reply:
speakin, of which, my copy/paste is wrong.  the readme has it under motion state, rebasing...